sigopt_sklearn/ensemble.py

```diff
@@ -47,27 +52,26 @@ def find_bayes_avg_coefs(self, X, y):
     z = z / np.sum(z)
     return z
 
-  def fit(self, X, y, client_token=None, cv=5, est_timeout=None,
-          cv_timeout=None, n_iter=25, n_jobs=1, n_cv_jobs=5):
+  def fit(self, X, y, client_token=None, est_timeout=None):
     self.n_outputs_ = 1
-    self.classes_ = np.unique(check_array(y, ensure_2d=False,
-                                          allow_nd=True, dtype=None))
-    #Store X and y data for workers to use
+    self.classes_ = np.array(np.unique(check_array(y, ensure_2d=False,
+                                                   allow_nd=True, dtype=None)))
+    # Store X and y data for workers to use
     with open(self.X_file.name, 'wb') as outfile:
       pickle.dump(X, outfile, pickle.HIGHEST_PROTOCOL)
     with open(self.y_file.name, 'wb') as outfile:
       pickle.dump(y, outfile, pickle.HIGHEST_PROTOCOL)
 
     sigopt_procs = []
     for build_args in self.estimator_build_args:
-      # run separaete python process for each estiamtor with timeout
+      # run separate python process for each estiamtor with timeout
       p = Popen(["timeout", str(est_timeout), "python", sklearn_fit.__file__,
            "--estimator", build_args['estimator'],
            "--X_file", build_args['X_file'], "--y_file", build_args['y_file'],
            "--client_token", client_token,
            "--output_file", build_args['output_file']])
       sigopt_procs.append(p)
-    exit_codes = [p.wait() for p in sigopt_procs]
+    exit_codes = [p.wait() for _ in sigopt_procs]
```

Comment:
err, no it won't. good catch. i think the linter got confused because of the reuse of p. will fix